nb-api/src/main/java/io/nosqlbench/api/metadata/SystemId.java

```diff
@@ -70,9 +72,22 @@ private static Optional<InetAddress> getMainInetAddrDirect() {
                 if (i2 instanceof Inet4Address) return 1;
                 return 0;
             }).findFirst();
-        return firstInetAddrForInterface;
-    }
 
+        if ( firstInetAddrForInterface.isPresent() &&
+             firstInetAddrForInterface.orElse(null).getHostAddress().contains(".")) return firstInetAddrForInterface;
```

Comment:
I know this PR is already merged, but do we really need the `orElse(null)` on here?

Reply:
Good point. I have a new PR almost ready to fix this.